package.json

```
@@ -4,7 +4,8 @@
     "test": "bun test",
     "fmt": "bun x prettier --plugin prettier-plugin-sh -w **/*.sh .sample/run.sh new.sh **/*.ts **/*.md *.md && terraform fmt **/*.tf .sample/main.tf",
     "fmt:ci": "bun x prettier --plugin prettier-plugin-sh --check **/*.sh .sample/run.sh new.sh **/*.ts **/*.md *.md && terraform fmt -check **/*.tf .sample/main.tf",
-    "lint": "bun run lint.ts"
+    "lint": "bun run lint.ts",
+    "update-version": "LATEST_TAG=$(git describe --abbrev=0 --tags | sed 's/^v//') && find . -name 'README.md' -exec sed -i \"s/\\( version \\s*=\\s*\\\"\\)[0-9]*\\.[0-9]*\\.[0-9]*\\(\\\"\\)/\\1$LATEST_TAG\\2/g\" {} \\;"
```

How about a regex that preserves whitespace?
```suggestion
		"update-version": "LATEST_TAG=$(git describe --abbrev=0 --tags | sed 's/^v/asdf/') || exit $?; replace() { find . -name README.md -exec sed \"$@\" -E 's/^([[:space:]]+version[[:space:]]+=[[:space:]]+\")[^\"]+(\")/\\1'$LATEST_TAG'\\2/' {} +; }; replace -i '' 2>/dev/null || replace -i"
```
For good measure, added cross-compat between macOS sed and GNU sed (`-i` behavior differs 😔). Perhaps better to use `awk` with temp file or something for the best compat, but `¯\_(ツ)_/¯`.

Also, the `find` change from `\;` to `+` is significant in that it will pass all files in one invocation, otherwise `find` might not return an error exit code and the `||` would not execute. Also more performant but that's not an issue here lol.

---

That is why I always want your advice on any shell script work. I would appreciate some help in making it bulletproof.